job_definitions/create_index.yml

```diff
@@ -0,0 +1,56 @@
+{% for environment in ['preview', 'staging', 'production'] %}
+- job:
+    name: "create-index-{{ environment }}"
+    display-name: "Create index - {{ environment }}"
+    project-type: freestyle
+    description: >
+      This job creates a new index in the ElasticSearch service from specified objects in the database.
+
+      Visiting either https://search-api.<STAGE>.marketplace.team/_status or
+      https://search-api.digitalmarketplace.service.gov.uk/_status will show you currently available indexes and their
+      aliases.
+
+      You can use the above to get an idea of naming conventions.
+
+      Once you have created your new index you might want to use it on the site. To do so, you can give your new index
+      an 'alias' here https://ci.marketplace.team/job/update-{{ environment }}-index-alias/
+    parameters:
+      - choice:
+          name: OBJECTS
+          choices:
+            - briefs
+            - services
+          default: services
+      - string:
+          name: INDEX_NAME
+          default: {{ search_config['services'][environment].default_index }}
```

Comment:
Would it be better to not supply a default at all? I'm guessing that most of the time these are going to be called by other Jenkins jobs, from where we can specify these details. Might help prevent an accident.

Reply:
Agreed.